src/Util/VerboseLogging.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc;
+
+namespace sodoff.Utils;
+
+public class VerboseLogging {
+    public static void log() {
+        Console.WriteLine("[INFO] Server is loading Assets");
```

Comment:
What assets are loaded at this time?

Logs should inform what the program is actually doing/starting to do/finished doing.

Reply:
i removed that because i need add a logger i had a idea to read from the api